packages/cli/commands/sandbox/sync.js

```diff
@@ -0,0 +1,298 @@
+          }),
+        },
+      ]);
+      if (!confirmed) {
+        process.exit(EXIT_CODES.SUCCESS);
+      }
+    }
+  } else if (accountConfig.sandboxAccountType === 'STANDARD') {
+    const standardSyncUrl = `${getHubSpotWebsiteOrigin(
+      getEnv(accountId) === 'qa' ? ENVIRONMENTS.QA : ENVIRONMENTS.PROD
+    )}/sandboxes-developer/${parentAccountId}/sync?step=select_sync_path&id=${parentAccountId}_${accountId}`;
+
+    logger.log(
+      i18n(`${i18nKey}.info.standardSandbox`, {
+        url: standardSyncUrl,
+      })
+    );
+    logger.log(
+      i18n(`${i18nKey}.info.sync`, {
+        parentAccountName: getAccountName(parentAccountConfig),
+        sandboxName: getAccountName(accountConfig),
+      })
+    );
+    uiLine();
+    logger.warn(i18n(`${i18nKey}.warning.standardSandbox`));
+    uiLine();
+    logger.log('');
+
+    if (!force) {
+      // Skip confirmation if force flag is present.
+      const { confirmSandboxSyncPrompt: confirmed } = await promptUser([
+        {
+          name: 'confirmSandboxSyncPrompt',
+          type: 'confirm',
+          message: i18n(`${i18nKey}.confirm.standardSandbox`, {
+            parentAccountName: getAccountName(parentAccountConfig),
+            sandboxName: getAccountName(accountConfig),
+          }),
+        },
+      ]);
+      if (!confirmed) {
+        process.exit(EXIT_CODES.SUCCESS);
+      }
+    }
+  } else {
+    logger.error('Sync must be run in a sandbox account.');
+    process.exit(EXIT_CODES.ERROR);
+  }
+
+  let initiateSyncResponse;
+
+  const baseUrl = getHubSpotWebsiteOrigin(
+    getEnv(accountId) === 'qa' ? ENVIRONMENTS.QA : ENVIRONMENTS.PROD
+  );
+
+  try {
+    logger.log('');
+    spinnies.add('sandboxSync', {
+      text: i18n(`${i18nKey}.loading.startSync`),
+    });
+
+    // Fetches sync types based on default account. Parent account required for fetch
+    const tasks = await getAvailableSyncTypes(
+      parentAccountConfig,
+      accountConfig
+    );
+
+    initiateSyncResponse = await initiateSync(
+      parentAccountId,
+      accountId,
+      tasks,
+      accountId
+    );
+
+    uiLine();
+    logger.log(
+      i18n(`${i18nKey}.info.syncStatus`, {
+        url: `${baseUrl}/sandboxes-developer/${parentAccountId}/development`,
+      })
+    );
+    uiLine();
+
+    logger.log('');
+    spinnies.succeed('sandboxSync', {
+      text: i18n(`${i18nKey}.loading.succeed`),
+    });
+  } catch (err) {
+    trackCommandUsage('sandbox-sync', { successful: false }, accountId);
+
+    spinnies.fail('sandboxSync', {
+      text: i18n(`${i18nKey}.loading.fail`),
+    });
+
+    logger.log('');
+    if (isMissingScopeError(err)) {
+      logger.error(
+        i18n(`${i18nKey}.failure.missingScopes`, {
+          accountName: getAccountName(parentAccountConfig),
+        })
+      );
+    } else if (
+      isSpecifiedError(
+        err,
+        429,
+        'RATE_LIMITS',
+        'sandboxes-sync-api.SYNC_IN_PROGRESS'
+      )
```

Comment:
New usage. Must be as specific as possible in case BE changes categories, or else we'll fall back to the basic logErrorInstance at the end